.github/workflows/lint.yaml

```diff
@@ -1,23 +1,28 @@
-name: Run linters/fixers
+---
+name: Lint
 on: [push]
+permissions: {}
 jobs:
-  shellcheck:
-    name: Shellcheck
+  superlinter:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: read
+      statuses: write
     steps:
     - uses: actions/checkout@v4
       with:
+        fetch-depth: 0
```

Comment:
I verified that it really does require this, even though we're not currently using any of the linters which use the commit history 🤷